sql-cli/sql_cli/constants.py

```diff
@@ -1,3 +1,5 @@
+STATE = {"debug": False}
```

Comment:
Since this value should change based on user configuration, it may make sense for us not to have it in `constants.py` but in a `settings.py` module, WDYT?